src/open_inwoner/accounts/views/cases.py

```diff
@@ -149,24 +169,66 @@ def get_context_data(self, **kwargs):
             context["case"] = {
                 "identification": case.identificatie,
                 "start_date": case.startdatum,
-                "end_date": case.einddatum if hasattr(case, "einddatum") else None,
+                "end_date": (case.einddatum if hasattr(case, "einddatum") else None),
                 "description": case.omschrijving,
-                "type_description": case_type.omschrijving
-                if case_type
-                else _("No data available"),
-                "current_status": statuses[-1].statustype.omschrijving
-                if statuses
-                and statuses[-1].statustype.omschrijving
-                in [st.omschrijving for st in status_types]
-                else _("No data available"),
+                "type_description": (
+                    case_type.omschrijving if case_type else _("No data available")
+                ),
+                "current_status": (
+                    statuses[-1].statustype.omschrijving
+                    if statuses
+                    and statuses[-1].statustype.omschrijving
+                    in [st.omschrijving for st in status_types]
+                    else _("No data available")
+                ),
                 "statuses": statuses,
-                "documents": case_info_objects,
+                "documents": documents,
             }
-            context["anchors"] = self.get_anchors(statuses, case_info_objects)
+            context["anchors"] = self.get_anchors(statuses, documents)
         else:
             context["case"] = None
         return context
 
+    def get_case_document_files(self, case) -> List[SimpleFile]:
+        case_info_objects = fetch_case_information_objects(case.url)
+
+        # get the information objects for the case objects
+
+        # TODO we'd like to use parallel() but it is borked in tests
```

Comment:
I've created a task to investigate this one - https://taiga.maykinmedia.nl/project/open-inwoner/task/919?